src/pages/who-we-are/_MakeTrading.tsx

```diff
@@ -38,9 +38,9 @@ const StyledFirstSectionText = styled(Header)`
 const MakeTrading = () => {
     return (
         <FirstSectionContainer padding="120px 0 80px" background="var(--color-white)">
-            {first_section_texts.map((item, index) => (
-                <StyledFirstSectionText as="p" type="unset" key={index} size="24px">
-                    {item.text}
+            {first_section_texts.map(({ text }) => (
+                <StyledFirstSectionText as="p" type="unset" key={+text} size="24px">
```

Comment:
Doesn't this return NAN if we try to coerce a text string to a number?

Reply:
oh yes that's right

Reply:
I didn't want to pass index as key